octoprint_mrbeam/__init__.py

```diff
@@ -1798,7 +1799,6 @@ def getHostname(self):
 				self._logger.warn("getHostname() system hostname got changed to: {}. Requires reboot to take effect!".format(hostname_dev_info))
 
 
-		return self._hostname
```

Comment:
This looks like removed by accident! If now, pls explain.

Reply:
sorry O:) thank you for pointing it out!!